api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/CompositionCohortQueries.java

```diff
@@ -124,7 +124,7 @@ public CompositionCohortDefinition getPatientBellowOneYearCurrentlyInARTStarted(
 		
 		// TODO -Check specifications document on how to use query 6,7 and 8
 		patientBellowOneYearCurrentlyInARTStarted.getSearches().put("9", new Mapped<CohortDefinition>(ageCohort, ParameterizableUtil.createParameterMappings("effectiveDate=${effectiveDate}")));
-		patientBellowOneYearCurrentlyInARTStarted.setCompositionString("((1 or 2 or 3 or 4) and (not 5)) and 9");
+		patientBellowOneYearCurrentlyInARTStarted.setCompositionString("((1 or 2 or 3 or 4) and (not 5) and not ( 6 and not (7 or 8))) and 9");
```

Comment:
Combing many cohortDefinitions in the same connection string might be confusing, it would be better to group individual cohort that  return common results and then combine them with others

Reply:
I agree ...It is confusing but is accurately reflecting/following [this section](https://docs.google.com/document/d/1g1xg7FVEBuaHpyFIt8vrt5pE2fMyuVJ5ie9OFND19_A/edit#bookmark=id.g0l8g2af3a7w) of the specifications which was helpful during the discussion with the BA team. Once we finish testing, I'll for sure group this definitions and use a more readable composition string.